### DISCUSSION TITLE
Поддержка SSR?

### DISCUSSION THREAD

**Discussion by MrFallen** [qruto/laravel-wave#18] [Q&A]
Добрый день, использую Laravel Jetstream (Vue + Inertia + SSR). Работает ли этот пакет с SSR? Или как можно заставить его работать с SSR?

**Comment by slavarazum** [comment]
Hello!
Wave is simply a driver for Laravel Echo, so its functionality should be quite similar to other drivers.
Echo focuses on real-time updates. What actually do you need in SSR environment?

**Comment by slavarazum** [comment]
If you have any specific concerns about using Echo and Wave, please reopen the discussion with details on the subject.
